projects/data/environment.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   - lalsuite
   - astropy
   - bilby>1.4,<2
+  - pycbc
```

Comment:
What is this used for?

Reply:
I was working on `pycbc` wf gen in another branch, let me remove it